src/utils/composeTree.js

```diff
@@ -25,4 +25,31 @@ const composeTree = (hierarchy, properties, filterType = null) => {
   return composeTreeRecursive(hierarchy, filterType);
 };
 
+/**
+ * Reorders tree to conform to a specific ordering.
+ *
+ * `order` is a list containing the property keys that we want to reorder.
+ *
+ */
+const setHierarchyOrder = (inputHierarchy, order = null) => {
+  if (!order) {
```

Comment:
I would just avoid calling this function if we don't have an order instead of allowing it to be null but not doing anything in that case.

Reply:
This function was left over when I was trying out different approaches to for cellSets ordering. It's not used anymore so I'll remove this,